config/webpack.dev.config.js

```diff
@@ -26,7 +26,8 @@ function generateWebpackDevConfig(buildOptions) {
         },
       ],
     },
-    hot: true,
+    hot: false,
```

Comment:
These have never worked, but with v4 they appear to actually be connecting to our site and attempting to work. Setting `hot` to false results in a couple second improvement in rebuild times. We could turn on live reload if people prefer it to be on.

Reply:
I'm so used to not having it I am all for leaving it off and getting faster rebuilds

Reply:
To be clear, live reloading doesn't add any time, but hot reloading does. Live reloading is just a thing that refreshes the page when you make a change. Hot reloading tries to directly replace a changed React component on your page without a refresh.

Reply:
ooohhh... thanks for clarifying that.